contracts/SSVNetwork.sol

```diff
@@ -654,6 +654,10 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         return (feeChangeRequest.fee.expand(), feeChangeRequest.approvalBeginTime, feeChangeRequest.approvalEndTime);
     }
 
+    function feeRecipientAddress(address recipientAddress) external override {
```

Comment:
not sure if this function should be called only from EOA or can be called from contracts too.
if its mandatory only for the former, then you can check:
```
if(msg.sender == tx.origin) {
    revert OnlyEOAAllowed();
}
```
cc @andrew-blox

Reply:
I dont think its needed here but thanks @mtabasco for this article it helped explain the concept to me:
https://ethereum.stackexchange.com/questions/113962/what-does-msg-sender-tx-origin-actually-do-why